next-docs/public/examples/switch/Rtl.tsx

```diff
@@ -1,21 +0,0 @@
-import React from 'react';
```

Comment:
We need Right To Left test case here, because we have arabic versions of our sites, please not remove it
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir